src/components/StaffList/index.jsx

```diff
@@ -33,17 +81,83 @@ class StaffList extends React.Component {
     }, () => this.props.input.onChange(this.state.staffList));
   }
 
+  /**
+   * The callback method for when a user selects a suggestion in the autocomplete list.
+   * Responsible for updating the search input text after selection and for updating the redux
+   * state of selected options.
+   *
+   * @param selectedValue - the selected autocomplete option
+   * @returns {string} - the value for the search input box after a selection
+   */
+  getSuggestionValue(selectedValue) {
+    const { value } = this.props.input;
+    const newInstructor = (selectedValue.id === 'new');
+    const instructorAlreadySelected =
+      (value.filter(staff => staff.uuid === selectedValue.text.uuid)).length > 0;
+    if (instructorAlreadySelected) {
+      // do we want to message the user?
+    }
+    if (!instructorAlreadySelected && !newInstructor) {
+      value.push(selectedValue.text); // add to form state
+    }
+    // clear out search input text
+    return '';
+  }
+
   handleRemove(uuid) {
     this.setState({
       staffList: this.state.staffList.filter(staffer => staffer.uuid !== uuid),
     }, () => this.props.input.onChange(this.state.staffList));
   }
 
+  /**
+   * Renders a suggestion in the autocomplete 'div'.  Called for each suggestion returned from
+   * the fetch response.
+   *
+   * @param suggestion - the suggestion object in the state.suggestions array
+   * @returns {*} - the rendered suggestion d
+   */
+  renderSuggestion(suggestion) {
+    return (
+      <div>
+        <table className="instructor-option" id={suggestion.id}>
+          <tbody>
+            <tr>
+              <td><img src={suggestion.text.profile_image_url} alt="profile url" className="rounded-circle" /></td>
+              <td>
+                <b>{suggestion.text.given_name} {suggestion.text.family_name}</b>
+                {suggestion.link && (
+                  <React.Fragment>
+                    <br />
+                    <a href={suggestion.link}>Add new Instructor</a>
+                  </React.Fragment>
+                )}
+                {suggestion.text.position && (
+                  <p>{suggestion.text.position.title} at {
```

Comment:
Nit: if you're not putting the full thing on line, you usually "completely nest it" for lack of a better term

i.e.
```html
<p>
  stuff
</p>
```

Reply:
Tried to do this within the limits of what the linter let me get away with.